pages/portfolio.html

```diff
@@ -4,9 +4,56 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>Portfolio</title>
+  <link href="../styles/portfolio-style.css" rel="stylesheet">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200;300;500&display=swap" rel="stylesheet">
 </head>
 <body>
+  <header>
+    <nav class="site-links">
+      <ul>
+        <li><a href="index.html">Home</a></li>
+        <li><a href="about.html">About me</a></li>
+      </ul>
+    </nav>
+  </header>
+
+  <section class="container">
+    <h1>Projects</h1>
+    <article>
+      <h2><a href="https://github.com/beaniebaye/viewing-party">Viewing Party</a></h2>
+      <p>First official python project! Practiced writing functions, introductory problem solving, nested data structures and test driven development.</p>
+    </article>
+    <article>
+      <h2><a href="https://github.com/beaniebaye/adagrams-py">Adagrams</a></h2>
+      <h3>In Collaboration with <a href="https://github.com/jacyyang04" class="partner">Jacy Yang</a></h3>
```

Comment:
This should probably just be a styled paragraph, not a `<h3>`.  The reason is because you wouldn't want to put `In Collaboration with`... as an entry in your table of contents:

```suggestion
      <p class="collaboration-note">In Collaboration with <a href="https://github.com/jacyyang04" class="partner">Jacy Yang</a></p>
```